README.md

```diff
@@ -17,13 +19,51 @@ For the latest information, please see the [New Relic docs](https://docs.newreli
 * Microsoft <a target="_blank" href="https://dotnet.microsoft.com/download/dotnet-framework">.NET Framework 4.5+</a> or  <a target="_blank" href="https://dotnet.microsoft.com/download/dotnet-core">.NET Core 2.0+</a>
 * <a target="_blank" href="https://docs.newrelic.com/docs/release-notes/agent-release-notes/net-release-notes">New Relic .NET Agent 8.21+<a>
 * <a target="_blank" href="https://docs.newrelic.com/docs/agents/net-agent/net-agent-api" target="_blank">New Relic .NET Agent API 8.21+</a>
-   
 
-## Open Source License
-This project is distributed under the [Apache 2 license](LICENSE).
+## Support
+
+Should you need assistance with New Relic products, you are in good hands with several support diagnostic tools and support channels.
 
+This [troubleshooting framework](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) steps you through common troubleshooting questions. 
 
-## Support
-New Relic has open-sourced this project. This project is provided AS-IS WITHOUT WARRANTY OR DEDICATED SUPPORT. Issues and contributions should be reported to the project here on GitHub.
+New Relic offers NRDiag, [a client-side diagnostic utility](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) that automatically detects common problems with New Relic agents. If NRDiag detects a problem, it suggests troubleshooting steps. NRDiag can also automatically attach troubleshooting data to a New Relic Support ticket.
```

Comment:
I don't know if NRDiag has any features in it specific to troubleshooting log enricher issues; it might be confusing to folks reading this if they assume mentioning NRDiag for this repo means that it does.

Reply:
I confirmed we should remove this